g2p/convert.py

```diff
@@ -16,12 +16,17 @@ class BlissLexiconToG2PLexiconJob(Job):
     Convert a bliss lexicon into a g2p compatible lexicon for training
     """
 
-    def __init__(self, bliss_lexicon):
+    __sis_hash_exclude = {"write_first_phoneme_only": True}
+
+    def __init__(self, bliss_lexicon, write_first_phoneme_only=True):
         """
         :param Path bliss_lexicon:
+        :param bool write_first_phoneme_only: In case of multiple phoneme representations for one lemma, when this is true it outputs only the first phoneme
         """
         self.bliss_lexicon = bliss_lexicon
 
```

Comment:
no newline between setting members from parameters (this should be one logical block)